src/common/components/DynamicComponentLoader.cpp

```diff
@@ -23,13 +21,23 @@ DynamicComponentLoader::~DynamicComponentLoader() {
         for (const auto &component: components) {
             ComponentFactory::getInstance().safeUnregisterComponent(component);
         }
+#ifdef _WIN32
```

Comment:
no windows 64 bit ?

Reply:
WIN32 also take windows 64 bit so it will work 👌